scripts-dev/update_debian_changelog.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -e
```

Comment:
should we make the title of this script a bit more indicative that it's specific to Docker?

Reply:
The instructions on how the script is meant to be run are in L16, which starts off by saying it's meant to be run in a container. Not sure how to be more verbose here.

Reply:
The name of the script seems very plain for it to actually turn out to require Docker

Reply:
(i.e. I would suggest `docker_update_debian_changelog.sh` but not sure if you can do better)

Reply:
Yeah I was thinking about doing something like that but didn't want people to misunderstand it as a script that is only needed when eg building docker images. But if you also think it could be worth renaming it that way then I'll do it.

Reply:
Hmm, I see your point. I think *something* would be nice though. 

I would probably be happy to call it `update_debian_changelog_in_docker.sh`. I also guess that docker image-specific scripts would live in `synapse/docker` so hopefully it's enough; thoughts?

It also means we can have a non-docker equivalent in the future if we so wished, without awkward naming issues.